models/ed/R/write.configs.ed.R

```diff
@@ -506,6 +506,9 @@ write.config.jobsh.ED2 <- function(settings, run.id) {
 
   jobsh <- gsub("@START_DATE@", settings$run$start.date, jobsh)
   jobsh <- gsub("@END_DATE@", settings$run$end.date, jobsh)
+  
+  jobsh <- gsub("@PFTNAMES@", sapply(settings$pfts, `[[`, "name"), jobsh)
+  jobsh <- gsub("@DHBREAKS@", unlist(settings$pfts$dbh.breaks), jobsh)
```

Comment:
note to self: this needs changing, `gsub` uses the first element only

Reply:
Is this supposed to be DBHBREAKS not DHBREAKS?

Since this seems like an optional arg, it would be good to first check that settings$pfts$dbh.breaks exists before doing this substitution, and having a sensible alternative if it's not specified

A change like this requires an update to the documentation